rust/examples/svc-simulcast.rs

```diff
@@ -0,0 +1,522 @@
+use actix::prelude::*;
+use actix_web::web::{Data, Payload};
+use actix_web::{web, App, Error, HttpRequest, HttpResponse, HttpServer};
+use actix_web_actors::ws;
+use mediasoup::prelude::*;
+use mediasoup::worker::{WorkerLogLevel, WorkerLogTag};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::num::{NonZeroU32, NonZeroU8};
+
+/// List of codecs that SFU will accept from clients
+fn media_codecs() -> Vec<RtpCodecCapability> {
+    vec![
+        RtpCodecCapability::Audio {
+            mime_type: MimeTypeAudio::Opus,
+            preferred_payload_type: None,
+            clock_rate: NonZeroU32::new(48000).unwrap(),
+            channels: NonZeroU8::new(2).unwrap(),
+            parameters: RtpCodecParametersParameters::from([("useinbandfec", 1_u32.into())]),
+            rtcp_feedback: vec![RtcpFeedback::TransportCc],
+        },
+        RtpCodecCapability::Video {
+            mime_type: MimeTypeVideo::Vp8,
+            preferred_payload_type: None,
+            clock_rate: NonZeroU32::new(90000).unwrap(),
+            parameters: RtpCodecParametersParameters::default(),
+            rtcp_feedback: vec![
+                RtcpFeedback::Nack,
+                RtcpFeedback::NackPli,
+                RtcpFeedback::CcmFir,
+                RtcpFeedback::GoogRemb,
+                RtcpFeedback::TransportCc,
+            ],
+        },
```

Comment:
Unless you enable VP9 here, you'll not be able to test VP9 SVC

Reply:
added